docs/developers/contributing.md

```diff
@@ -29,7 +29,7 @@ Bugfixes should include a regression test which prevents that bug from being re-
 
 ## Update Documentation
 
-Documentation for the project is stored in the [docs](..) directory. If your feature requires docs updates,
+Documentation for the project is stored in the [docs](https://github.com/prebid/prebid.github.io/tree/master/prebid-server) directory. If your feature requires docs updates,
```

Comment:
I think we should also point to the prebid.org URL that has the final, pretty version of the docs. Keep this link so people know where to go if anything needs to be added or edited.